chart/values.yaml

```diff
@@ -51,6 +51,11 @@ base:
   default_req_timeout: 5s
   # -- Cache timeout for core agent & diskpool deployment
   cache_poll_period: 30s
+  logging:
+    # -- Valid values for format are pretty, json and compact
+    format: pretty
+    color: true
+    silenceLevel:
```

Comment:
Where are we setting the silenceLevel? Did not see that on the template.

Reply:
Not sure, Installation works post removing silenceLevel also. -  logSilenceLevel: , Doesnt have any further key/val also. Seems like an incomplete config.

@niladrih , @tiagolobocastro any ideas?